src/main/java/io/cloudnative/teamcity/WebhooksSettings.java

```diff
@@ -26,7 +26,7 @@
   Map<String,Set<String>> urls;
 
   public WebhooksSettings(@NonNull ServerPaths serverPaths) {
-    settingsFile = new File(serverPaths.getConfigDir(), SETTINGS_FILE);
+    settingsFile = newFile(serverPaths.getConfigDir(), SETTINGS_FILE);
```

Comment:
for testing purposes -
or else it's tricky to mock creation of an object in the constructor